sentry_sdk/scope.py

```diff
@@ -152,7 +160,8 @@ def _drop(event, cause, ty):
                     return _drop(event, processor, "error processor")
                 event = new_event
 
-        for processor in self._event_processors:
+        for processor in chain(self._event_processors, global_event_processors):
```

Comment:
Shouldn't it be `chain(global_event_processors, self._event_processors)`?

Reply:
I imagine that every event processor looks like this:

```
if "key" not in event:
    event["key"] = "bar"
```

and not like this:

```
event["key"] = "bar"
```

In the first example, the first event processor sets `"key"` and every other event processor skips their logic. So the *first processor wins*.

In the second example the *last processor wins*, as its the only one whose value does not get overridden.

The reason is that scopes are applied to the event before event processors are, and data set on the scope should win over data set by integrations.

I wonder if we should apply event processors to the event before we apply scope data.